topics/lists.md

```diff
@@ -17,49 +17,51 @@ body_class: lists
-| `week_created` | `INTEGER` | ISO 8601 week of when the list was created, note Monday is the start of the ISO 8601 week, values `1-53` | `52`
-| `dow_created` | `INTEGER` | Day of week of when the list was created, values `0-6`, Sunday is `0` | 5
-| `last_updated` | `TIMESTAMP` | Date the list was last updated | `2016-01-12 00:00:00.0`
-| `year_last_updated` | `INTEGER` | Year of when the list was last updated | `2018`
-| `month_last_updated` | `INTEGER` | Month of when the list was last updated, values `1-12` | `12`
-| `week_last_updated` | `INTEGER` | ISO 8601 week of when the list was last updated, note Monday is the start of the ISO 8601 week, values `1-53` | `52`
-| `dow_last_updated` | `INTEGER` | Day of week of when the list was last updated, values `0-6`, Sunday is `0` | 5
-| `last_published` | `TIMESTAMP` | Date the list was last published | `2016-01-12 00:00:00.0`
-| `year_last_published` | `INTEGER` | Year of when the list was last published | `2018`
-| `month_last_published` | `INTEGER` | Month of when the list was last published, values `1-12` | `12`
-| `week_last_published` | `INTEGER` | ISO 8601 week of when the list was last published, note Monday is the start of the ISO 8601 week, values `1-53` | `52`
-| `dow_last_published` | `INTEGER` | Day of week of when the list was last published, values `0-6`, Sunday is `0` | 5
-| `date_archived` | `TIMESTAMP` | Date the list was archived | `2016-01-12 00:00:00.0`
-| `year_archived` | `INTEGER` | Year of when the list was archived | `2018`
-| `month_archived` | `INTEGER` | Month of when the list was archived, values `1-12` | `12`
-| `week_archived` | `INTEGER` | ISO 8601 week of when the list was archived, note Monday is the start of the ISO 8601 week, values `1-53` | `52`
-| `dow_archived` | `INTEGER` | Day of week of when the list was archived, values `0-6`, Sunday is `0` | 5
-| `last_reviewed` | `TIMESTAMP` | Date the list was last reviewed | `2016-01-12 00:00:00.0`
-| `year_last_reviewed` | `INTEGER` | Year of when the list was last reviewed | `2018`
-| `month_last_reviewed` | `INTEGER` | Month of when the list was last reviewed, values `1-12` | `12`
-| `week_last_reviewed` | `INTEGER` | ISO 8601 week of when the list was last reviewed, note Monday is the start of the ISO 8601 week, values `1-53` | `52`
-| `dow_last_reviewed` | `INTEGER` | Day of week of when the list was last reviewed, values `0-6`, Sunday is `0` | 5
-| `privacy_control` | `VARCHAR(255)` | `Private` or `Public` | `Public`
-| `student_numbers` | `INTEGER` | Numbers of students | `84`
-| `default_list_view` | `VARCHAR(255)` | whether the list is using the new list view. 'Beta' refers to the new list view. 'Classic' refers to the old list view. | `Beta is default`
+| Column Name            | Datatype        | Description                                                                                                             | Example                                |
```

Comment:
[comment] I've reformatted the markup so that it is easier to edit in future. This has had the knock on effect of making the diff look bigger than it is.